### ISSUE TITLE
Placeholder issue #_FIRSTNAME

### ISSUE THREAD

**Issue by kevin-vanb** [liedekef/events-made-easy#400]
**Describe the bug**
We used to send emails for Christmas and New Year in the past with EME, personalised with #_FIRSTNAME as placeholder, but the field is not displayed in the mail not in subject and not in body.

**To Reproduce**
Go to email management
Send generic mail
Use a template
Enter a test recipient
Email is received succesfully but not adressed with firstname as expected

**Expected behavior**
Hello #_FIRSTNAME (with actual placeholder)

**Screenshots**

**Online Demo**
While screenshots can say something, they don't say everything.
So, if possible, include a link to a demo page demonstrating the issue.

**Log output**
If applicable, check your php error logfile and add php errors here

**Version info**
PHP version:
EME version:

**Additional context**

**Comment by liedekef** [comment]
Which is your EME version? I tried a generic mail with test recipient here and it works fine (dev code though)

**Comment by kevin-vanb** [comment]
Currently working on EME Version 2.4.21, PHP 8.2, strange thing is when I use #_LASTNAME, it works in that way that he displays the full name (Last and first name), so not 100% correct, but he displays the content at least :-)

Grtz
Kevin

**Comment by liedekef** [comment]
Is this for EME users linked to a WP user or "standalone" EME users?

**Comment by kevin-vanb** [comment]
They are linked to wordpress users indeed

**Comment by liedekef** [comment]
Well, I've just tried those too (and version 2.4.21): no issue. Can you reproduce this on a test setup and give me access? Do those people actually have a firstname in EME (not WP)?

**Comment by kevin-vanb** [comment]
Hello Franky, thanks for the tip.
Complex story :-)
We are a small club of hobbyists and both the membership administration as the events are done with EME, great for us :-)
But the event handling is actually separated from the members, you dont't need to be a member to attend to an event, but most of our members also forget to login to their wordpress account to create a registration to an event and due to this, the database gets polluted as I just saw (thanks to your tip)

So in short, EME works fine, ticket can be closed, problem exists in the duplicate entries and as with my test, Apologies for the search work and happy 2024 already, keep up the good work!

**Comment by kevin-vanb** [comment]
Problem due to duplicate entries in the backend, no issue with EME

**Comment by liedekef** [comment]
EME provides an integrity check (in the people overview) to help you detect/clean duplicates. Also I try my best to avoid duplicates :-)

**Comment by kevin-vanb** [comment]
Great, didn't see it before! Will work my way down the duplicates today :D
